src/components/IPFS/Config.js

```diff
@@ -0,0 +1,19 @@
+const Temporal = {
```

Comment:
Change this variable name to all caps `TEMPORAL` as it's a bunch of unchanging constants. And change the file name to all lowercase. Capitalised file names denote React components in our repository